lib/senkyoshi/models/content.rb

```diff
@@ -45,20 +53,23 @@ def iterate_xml(xml, pre_data)
       end
       @type = xml.xpath("/CONTENT/RENDERTYPE/@value").first.text
       @parent_id = pre_data[:parent_id]
-      bb_type = xml.xpath("/CONTENT/CONTENTHANDLER/@value").first.text
-      bb_type.slice! "resource/"
-      @module_type = CONTENT_TYPES[bb_type]
+      @module_type = self.class.name.gsub("Senkyoshi::", "")
```

Comment:
I am doing this because I am changing Attachment files to Wikipages if their files don't exist in the export. And so this is essentially just saying what is the class name minus the Senkyoshi:: part. 

The only one we are changing thus far is the quizzes to be "Quiz:Quizzes" because that can't be a class name. So maybe we would do it similarly to the way we are changing the quiz name to be the correct canvas module?